pontoon/base/static/css/style.css

```diff
@@ -251,21 +251,6 @@ button {  width: auto; overflow: visible; }
   color: #FED271;
 }
 
-.untranslated .status.fa:before {
-  color: #AAAAAA;
-  content: "";
-}
-
-.has-suggestions .status.fa:before {
-  color: #AAAAAA;
-  content: "";
-}
-
-.unchanged .status.fa:before {
-  color: #AAAAAA;
-  content: "";
-}
-
```

Comment:
I don't think we need these in the general CSS file. They are already defined in the translate.css.